examples/2016-10-31/lambda_edge/README.md

```diff
@@ -0,0 +1,40 @@
+## Lambda@Edge sample
+
+This example leverages [SAM Safe Deployments feature](https://github.com/awslabs/serverless-application-model/blob/master/docs/safe_lambda_deployments.rst) in order to ease Lambda@Edge deployments by automatically publishing a new version upon deployments. Since SAM supports standard Cloudformation resources Cloudfront Distribution configuration will be automatically updated as soon as a new Lambda function version is available.
+
+The following Lambda function snippet uses ``AutoPublishAlias`` property which provides an additional property named `<LogicalName>.Version`:
+
+```yaml
+LambdaEdgeFunctionSample:
+    Type: AWS::Serverless::Function
+    Properties:
+        CodeUri: s3://<bucket>/lambda_edge.zip
+        Runtime: nodejs6.10
+        Handler: app.handler
+        Timeout: 5
+        # More info at https://github.com/awslabs/serverless-application-model/blob/master/docs/safe_lambda_deployments.rst
+        AutoPublishAlias: live 
+```
+
+We can now configure our [Cloudfront Distribution Lambda Association Property](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cloudfront-distribution-lambdafunctionassociation.html) to always reference the latest available Lambda Function Version ARN:
+
+```yaml
+CFDistribution:
+    Type: AWS::CloudFront::Distribution
+    Properties:
+    DistributionConfig:
```

Comment:
The DistributionConfig key should be one more indention in.

```
CFDistribution:
  Type: AWS::CloudFront::Distribution
  Properties:
    DistributionConfig:
      Enabled: 'true'
      ...
```

Reply:
good catch! I was trying to find out why that was the case as I deployed everything to test, but... that was in the README ;)

Fixed in a new commit!